### PR TITLE
homepage initial setup

### PR DESCRIPTION
**Pull request by Icicleta**
### What change does this PR introduce?
- Creation of view file.
- Created homepage view.
- Routering homepage.

### Notes
- Not sure what should we put on title, just "homepage initial setup" doesn't seem enough.
- Should we decide if we stick to single quotes or double quotes?

### Ticket

- [Trello Ticket](https://trello.com/c/CM9aUu4r)

### CODE REVIEW

File: listings_app/urls.py [hunk 1]
```diff
@@ -16,6 +16,9 @@
 from django.contrib import admin
 from django.urls import path
 
+from . import views
+
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('', views.homepage, name="homepage")
```
Comment (amyboyd) [inline listings_app/urls.py]: This is a minor thing, but it is best to be consistent with formatting. If a file uses single quotes, you should try to use single quotes elsewhere in the file. You have mixed single and double quotes. See PEP8's recommendation here: https://www.python.org/dev/peps/pep-0008/#string-quotes

File: listings_app/views.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+from django.http import HttpResponse
+
+
+def homepage(request):
+    return HttpResponse("<h1>Hello! I am the homepage</h1>")
```
Comment (amyboyd) [inline listings_app/views.py]: Same as above.

This is a minor thing, but it is best to be consistent with formatting. If a file uses single quotes, you should try to use single quotes elsewhere in the file. You have mixed single and double quotes. See PEP8's recommendation here: https://www.python.org/dev/peps/pep-0008/#string-quotes

File: listings_app/urls.py (hunk 1 above)
Comment (Icicleta) [reply listings_app/urls.py]: Thanks @amyboyd, I thought about it, actually, I commented in the PR about stick to one of them but I didn't know which one, but you right, didn't realise that file was already on single quotes. I will fix :)

**Comment by Icicleta** [comment]
fixed @amyboyd

**Comment by lili2311** [comment]
👋 to answer your question about the app, yes we would normally make an app for everything in Django. So perhaps you can so that in the next PR? I think the app can be called `core` for all core pages or something like that?

File: listings_app/urls.py [hunk 3]
```diff
@@ -16,6 +16,9 @@
 from django.contrib import admin
 from django.urls import path
 
+from . import views
+
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('', views.homepage, name='homepage')
```
Comment (lili2311) [inline listings_app/urls.py]: normally the path would be `/`

Comment (Icicleta) [reply listings_app/urls.py]: hey @lili2311, thanks for your comments. About '/', I  thought the same, but when I added it I was said by VS Code that it was deprecated, that it wasn't needed anymore. Do you want me to add it anyway?

**Comment by Icicleta** [comment]
@lili2311, I have created the app core and moved the homepage to this app.

**Pull request merged**